chia/wallet/puzzles/nft_ownership_layer.clvm.hex

```diff
@@ -0,0 +1 @@
+expected defun, defun-inline, defmacro, or defconstant
```

Comment:
Looks like this file isn't present (was removed or missing)?  Probably should synchronize the hex and sha256.

Reply:
Looks like @trepca removed the content in https://github.com/Chia-Network/chia-blockchain/commit/4df1f4658d9d29704c7e876aa8913fe5863344ac

Reply:
@xdustinface indeed, I think it slipped while resolving conflicts, will fix, thanks